setup.py

```diff
@@ -95,7 +96,7 @@
             'invenio-app-rdm = invenio_app.cli:cli',
         ],
         'invenio_base.blueprints': [
-            'invenio_app_rdm = invenio_app_rdm.theme.views:ui_blueprint',
+            "invenio_app_rdm = invenio_app_rdm.theme.views:ui_blueprint",
```

Comment:
probably unnecessary ;)

Reply:
:smile: 
yes, that's still a left-over from experiments with multiple blueprints, and now it's just there looking awesome